appengine/cloud-tasks/create_task.rb

```diff
@@ -23,10 +23,10 @@
 # @param [Integer] seconds The delay, in seconds, to process your task.
 def create_task project_id, location_id, queue_id, payload: nil, seconds: nil
   # Instantiates a client.
-  cloud_tasks = Google::Cloud::Tasks.new
+  client = Google::Cloud::Tasks.new
```

Comment:
What was the rationale for this change? I'm somewhat partial to `cloud_tasks`

Reply:
I thought it fit the rubric better by naming it client. Would you prefer `cloud_tasks_client`?

Reply:
Yeah. Up to you. We haven't been super consistent on this.